penlight-scm-3.rockspec

```diff
@@ -1,14 +1,14 @@
 package = "penlight"
-version = "scm-2"
+version = "scm-3"
```

Comment:
You can't bump this without changing the file name (yes that's nuts, but...).

Reply:
huh? I did change the filename?

Reply:
Hmm, the diff viewer on Github proper was definitely not showing that. The one it their mobile app that I'm on this moment is. Don't know what to  say. I wonder if I have a user script on the computer I was on that interfered with the view, maybe because Github moved cheese. I'll check next time I'm on it.